src/services/OswServiceTasks.cpp

```diff
@@ -0,0 +1,45 @@
+#include "services/OswServiceTasks.h"
+
+#include "services/OswServiceTaskBLECompanion.h"
+#include "services/OswServiceTaskExample.h"
+#include "services/OswServiceTaskGPS.h"
+#include "services/OswServiceTaskMemMonitor.h"
+
+// Define helpers - as some are just defines and do not represent a numeric value (which is needed for the service amount calc)
+#ifdef GPS_EDITION
+    # define SERVICE_GPS 1
+#else
+    # define SERVICE_GPS 0
+#endif
+#ifdef DEBUG
+    # define SERVICE_MEMMONITOR 1
+#else
+    # define SERVICE_MEMMONITOR 0
+#endif
+
+namespace OswServiceAllTasks {
+#if SERVICE_BLE_COMPANION == 1
+OswServiceTaskBLECompanion bleCompanion;
+#endif
+#ifdef GPS_EDITION
+OswServiceTaskGPS gps;
+#endif
+//OswServiceTaskExample example;
+#ifdef DEBUG
+OswServiceTaskMemMonitor memory;
+#endif
+}  // namespace OswServiceAllTasks
+
+const unsigned char oswServiceTasksCount = 0 + SERVICE_BLE_COMPANION + SERVICE_GPS + SERVICE_MEMMONITOR;
```

Comment:
I was thinking about how we would do this one 👍 
too simple :)

Reply:
:grin: - is this comment done?